components/case-studies/case-studies.module.scss

```diff
@@ -5,7 +5,7 @@
   background-color: $color-bg-grey;
   display: flex;
   flex-direction: column;
-  padding: 7rem 0;
+  padding: 7rem 3rem;
```

Comment:
This change should not be introduced as this now causes the mobile breakpoint to deviate from the design, as seen below:

<img width="604" alt="image" src="https://user-images.githubusercontent.com/27775658/159674070-0000f9b1-76de-4e49-951e-67fdba1e3a94.png">

@rashidakanchwala I wonder why is this change introduced?